tests/playwright/src/model/pages/run-image-page.ts

```diff
@@ -81,7 +94,25 @@ export class RunImagePage extends BasePage {
       throw Error(`Start Button not enabled: ${errMessage}`);
     }
     await this.startContainerButton.click();
-    return new ContainersPage(this.page);
+    // After clicking on the button there seems to be four possible outcomes
+    // 1. Opening particular container's details page with tty tab opened
+    // 2. Opening Containers page with new container on it
+    // 3. staying on the run image page with an error
+    // 4. Starting a container without entrypoint or command creates a container, but it stays on Run Image Page without error
+    await waitWhile(
```

Comment:
Why not replace this waitWhile() with `await playExpect(this.name).toBeVisible()` ? I'm not clear what happens if the field is not visible in your current implementation before the timer runs out, since the error is not propagated, it seems to me that the test would continue even though the field wouldn't be visible.